tests/integration/targets/ssh_key_info/tasks/main.yaml

```diff
@@ -4,11 +4,7 @@
         r: "{{ 1000000000 | random }}"
         ssh_pub_key: "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQCloBr83q7v2/kipfpChnN1cPr+DA++JN3RdLBid3tFjy5UmAelpRmKO9Uhija7ElC9/x187t13smc35NGaDOG+s+zrSMlSfygqA6pf8njEPDRBhvyp12Fjwb8i+ILFXiU1GYaVprbQwpgdApwqieoCZ4pathc/HdPvznX/Aqgyiq+5+dSRa5GUftAPWrt3ScFXttlfihU+0wIanyxoxnrtPcqNshs39dRg8UP2zrB5aK+9nPurO/6qSWDqVnIVlparlqXxZdwKl+Gfiq93pGicrPgVEy49Tbl75Y8Nxj7zDsJsQuO0UlRk9IUoe1asAy5DXqQ6foOb6vR9rld6owqRHCG/tjYDpuw1uYCYj5xm+DDfkNRowWJktIgnajzaYqZ3ytWzg31m8a7X2Wol8foxlFrUqqqesJ05I28A3JvLIU5s4vSzi1hK4cWKSi15UFdxX1f1a+pRsRgY446k+i3uWcFZVybzp5tDRdjUMRO6XQrbjrxFnKurRY0+S1NeRF0="
 
-    - linode.cloud.api_request:
-        api_token: '{{ api_token }}'
-        ua_prefix: '{{ ua_prefix }}'
-
-        path: profile/sshkeys
+    - linode.cloud.api_request:        path: profile/sshkeys
```

Comment:
Should `path: profile/sshkeys` be in the next line?

Reply:
Oops looks like I missed that. I'll fix that real quick 👍